src/components/silentBanner/SilentBanner.js

```diff
@@ -0,0 +1,150 @@
+import Box from '../box';
+import Button from '../button';
+import ButtonGroup from '../buttonGroup';
+import Icon from '../icon';
+import { IconButton } from '../iconButton';
+import { Heading3, TextBody } from '../typography';
+
+const backgroundColorMap = {
+  error: 'ruby',
+  info: 'neutral',
+  success: 'mint',
+  warning: 'gold',
+};
+
+const backgroundTintMap = {
+  error: 'normal',
+  info: 'normal',
+  success: 'light',
+  warning: 'light',
+};
+
+const iconMap = {
+  error: IconWarningBadgedMediumFilled,
+  info: IconBellMediumOutline,
+  success: IconCheckmarkBadgedMediumFilled,
+  warning: IconWarningBadgedMediumFilled,
+};
+
+const iconColorMap = {
+  error: 'neutral',
+  info: 'neutral',
+  success: 'neutral',
+  warning: 'gold',
+};
+
+const iconTintMap = {
+  error: 'lightest',
+  info: 'darkest',
+  success: 'lightest',
+  warning: 'dark',
+};
+
+class SilentBanner extends PureComponent {
+  render() {
+    const {
+      children,
+      onClose,
+      primaryAction,
+      primaryActionLabel,
+      secondaryAction,
+      secondaryActionLabel,
+      showIcon,
+      status,
+      title,
+      ...others
+    } = this.props;
+
+    const hasActions = Boolean(primaryAction || secondaryAction);
+    const IconToRender = iconMap[status];
+
+    return (
+      <Box {...others} display="flex">
+        {status && (
+          <Box
+            backgroundColor={backgroundColorMap[status]}
+            backgroundTint={backgroundTintMap[status]}
+            borderBottomLeftRadius="rounded"
+            borderTopLeftRadius="rounded"
+            paddingHorizontal={showIcon ? 2 : 1}
+            paddingVertical={4}
+          >
+            {showIcon && (
+              <Icon color={iconColorMap[status]} tint={iconTintMap[status]}>
+                <IconToRender />
+              </Icon>
+            )}
+          </Box>
+        )}
+        <Box
+          borderColor="neutral"
+          borderTint="normal"
+          borderLeftWidth={status ? 0 : 1}
+          borderTopWidth={1}
+          borderRightWidth={1}
+          borderBottomWidth={1}
+          borderBottomRightRadius="rounded"
+          borderTopRightRadius="rounded"
+          borderBottomLeftRadius={status ? 'square' : 'rounded'}
+          borderTopLeftRadius={status ? 'square' : 'rounded'}
+          display="flex"
+        >
+          <Box padding={4}>
+            {title && (
+              <Heading3 color="teal" marginBottom={2}>
+                {title}
+              </Heading3>
+            )}
+            {children && <TextBody color="teal">{children}</TextBody>}
+            {hasActions && (
+              <ButtonGroup display="flex" justifyContent="flex-end" marginTop={4}>
+                {secondaryAction && (
+                  <Button level="link" size="small">
+                    {secondaryActionLabel}
+                  </Button>
+                )}
+                {primaryAction && <Button size="small">{primaryActionLabel}</Button>}
```

Comment:
Not very handy if we can only set the label 😉 

You can use the same approach as in the Dialog component.

```js
{secondaryAction && <Button level="link" size="small" {...secondaryAction} />}
{primaryAction && <Button level="primary" size="small" {...primaryAction} />}
```

This way you also don't need a separate prop (primaryAction & primaryActionLabel)